session/types.proto

```diff
@@ -85,7 +85,23 @@ message SessionToken {
   neo.fs.v2.refs.Signature signature = 2 [json_name = "signature"];
 }
 
-// Extended headers for Request/Response.
+// Extended headers for Request/Response. May contain any user-defined headers
+// to be interpreted on application level.
+//
+// Key name must be unique valid UTF-8 string. Value can't be empty. Requests or
+// Responses with duplicated header names or headers with empty values will be
+// considered invalid.
+//
+// There are some "well-known" headers starting with `__NEOFS__` prefix that
+// affect system behaviour:
+//
+// * __NEOFS__NETMAP_EPOCH \
+//   Netmap epoch to use for object placement calculation. If set to '0' or not
```

Comment:
It would be nice to specify format of the string value for both `NETMAP_EPOCH` and `NETMAP_LOOKUP_DEPTH`. E.g. add `Value represented as decimal number` lines.